lib/space.ts

```diff
@@ -42,10 +42,13 @@ const spaceMethods = [
 
   'getAllScheduledActions',
   'getEntityScheduledActions'
-]
+] as const
 
-module.exports = function createSpace(channel, initialContentTypes) {
-  const space = {}
+export default function createSpace(channel, initialContentTypes) {
+  const space = {} as {
```

Comment:
Maybe we can define an interface here?

Reply:
Nice catch. There was already a `SpaceAPI` interface for me to use :slightly_smiling_face: